documentation/book/proc-user-operator-with-resource-requests-limits.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// assembly-deploying-the-user-operator.adoc
+
+[id='proc-user-operator-with-resource-requests-limits-{context}']
+= Configuring the User Operator with resource requests and limits
+
+.Prerequisites
+
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `Kafka` resource specifying in the `Kafka.spec.entityOperator.userOperator.resources` property the resource requests and limits you want the User Operator to have.
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+spec:
+  # kafka and zookeeper sections...
+  userOperator:
```

Comment:
You should add the `entityOperator` here.

Reply:
Agree. I will change the proc for the topic Operator as well then.

Reply:
You should check the context for the TopicOperator - It would be correct like this for TO if you use the deprecated TO outside of EO. So maybe you would need to change it on more places there.

Reply:
The procedure is used only for the TO inside the EO (not with the deprecated one)